examples/31_minimal/src/entries.tsx

```diff
@@ -1,27 +1,24 @@
-import { defineEntries } from 'waku/server';
-import { Slot } from 'waku/client';
+import {
+  new_defineEntries,
+  unstable_renderRsc as renderRsc,
+  unstable_decodeRscPath as decodeRscPath,
+} from 'waku/minimal/server';
 
 import App from './components/App';
 
-export default defineEntries(
-  // renderEntries
-  async (rscPath) => {
-    return {
-      App: <App name={rscPath || 'Waku'} />,
-    };
-  },
-  // getBuildConfig
-  async () => [{ pathname: '/', entries: [{ rscPath: '' }] }],
-  // getSsrConfig
-  async (pathname) => {
-    switch (pathname) {
-      case '/':
-        return {
-          rscPath: '',
-          html: <Slot id="App" />,
-        };
-      default:
-        return null;
+export default new_defineEntries({
+  unstable_handleRequest: async (config, ctx) => {
+    const basePrefix = config.basePath + config.rscBase + '/';
+    if (ctx.req.url.pathname.startsWith(basePrefix)) {
+      const rscPath = decodeRscPath(
+        decodeURI(ctx.req.url.pathname.slice(basePrefix.length)),
+      );
+      ctx.res.body = renderRsc(config, ctx, {
+        App: <App name={rscPath || 'Waku'} />,
+      });
```

Comment:
This might feel too low-level (it's closer to middleware api), but I can't come up with other ideas (one idea with ALS is dropped).

Reply:
what is ALS?

Reply:
AsyncLocalStorage

Reply:
The API is improved since this. It now hides ctx and config.